publish.py

```diff
@@ -14,6 +14,7 @@
     'crates/cargo-platform',
     'crates/cargo-util',
     'crates/crates-io',
+    'crates/home',
```

Comment:
@ehuss, I wonder whether we need to notify Release team for this change.

Reply:
I didn't include this change in the second version since I don't think we should release home crate when we release cargo and this is the cargo release script I assume that runs when we do cargo specific releases?